webui/react/src/components/NavigationTabbar.tsx

```diff
@@ -79,6 +85,19 @@ const NavigationTabbar: React.FC = () => {
       </div>
       <ActionSheet
         actions={[
+          {
+            render: () => {
+              return <AvatarCard className={css.user} user={auth.user} />;
+            },
+          },
+          {
+            label: 'Settings',
```

Comment:
`non-blocking` Might be nice for the sake of consistency to have icons on the bottom menu for settings and sign out, maybe `settings` and `user` respectively? The options currently available are in the `icons.scss` file but there's a possibility to add more as well.